### PR TITLE
match @types with ts3.6

### PR DESCRIPTION
**Pull request by piwysocki**

### CODE REVIEW

**Comment by adrianmroz-allegro** [comment]
Plywood bump require some local tests from @allegro/turnilo - stay tuned.

**Comment by adrianmroz-allegro** [comment]
I tried to rebase this PR in #1036 but got a lot of type errors. Probably we need to revisit this.

**Comment by piwysocki** [comment]
> I tried to rebase this PR in #1036 but got a lot of type errors. Probably we need to revisit this.

Hi, Can you try run lint and tests on this branch?

**Comment by piwysocki** [comment]
let me update with ts3.6

**Comment by piwysocki** [comment]
done

**Comment by piwysocki** [comment]
Hi @adrianmroz-allegro can you help with "Size limit" error?

**Comment by adrianmroz-allegro** [comment]
> Hi @adrianmroz-allegro can you help with "Size limit" error?

That's some issue with permissions. Can't help right now, but from logs I can see that sizes didn't change so I'll override and merge ;)

File: src/server/utils/file/file.ts [hunk 1]
```diff
@@ -24,6 +24,6 @@ export function loadFileSync(filepath: string, postProcess: "json" | "yaml"): ob
     case "json":
       return JSON.parse(fileData);
     case "yaml":
-      return yaml.safeLoad(fileData);
+      return yaml.load(fileData) as object;
```
Comment (adrianmroz-allegro) [inline src/server/utils/file/file.ts:27]: Note for future self: https://github.com/nodeca/js-yaml/blob/ab31bba6b41f58390f431123ffec5031b986edf5/migrate_v3_to_v4.md#safeload-safeloadall-safedump--load-loadall-dump

**Review by adrianmroz-allegro — APPROVED** [review]

**Pull request merged**